Tasks/Trots/Task1/Task1.py

```diff
@@ -0,0 +1,12 @@
+from decimal import Decimal
+from decimal import getcontext
+
+getcontext().prec = 50
+
+start_deposit = Decimal('20000')
+rate = Decimal('0.15')
+term = Decimal('60')
+
+final_sum = start_deposit*(Decimal('1') + rate/Decimal('12'))**term
+
+print("The amount of the deposit at the end of the term will be - ", final_sum)
```

Comment:
and here we get a 50 decimal digits based answer) would be nice to round it to get some realistic money related answer (like with some cents)

Reply:
i just wanted to show the difference between using decimals and normal numbers so i didn't use round